### PR TITLE
Prisonconsole fixed

### PR DESCRIPTION
**Pull request by AffectedArc07**
# REMAKE OF #10943 
**What does this PR do:**
This PR adds in a new console for the brig, which allows you to view the status of all current brig cells. It looks like this.
image

When clicked, it will show this UI:
image
This has a list of all cell timers in the world, and their status. Status includes whether they are in use or not, and if they are, who is in there, who put them in there, what crimes they committed, how long they were brigged for, how long is left on their sentence, and a button to release them. 

**Images of sprite/map changes (IF APPLICABLE):**
New sprite (Thanks `Alonefromhell`:
image

# Map edits:
Wardens office:
image

Front Desk:
image

HoS office:
image

Few ideas for things, which will be added if asked:
Maybe an emag act which forces open all cells?
Maybe a log of previous brigs on a console instead of the sheets of paper?

**Changelog:**
:cl: AffectedArc07 and Alonefromhell
add: Cell management consoles: See the status of all brig cells without having to leave your office
imageadd: New screen sprite for cell management console
/:cl:

# Mapmerge
I think its just straight up broken. I did it on my previous PR and it didnt work, and I did it on this PR and it didnt work. Its not computer-related as I am using a different PC here than I used to make my previous PR. The log is below if it makes any sense
```
File to use: 0
Loading map cyberiad.dmm.backup
Loading tiles 8992
Loading map cyberiad.dmm
Loading tiles 8992
Loading levels
New map part from (1,1,1)
We have 8992 different tiles
Done
```

### CODE REVIEW

**Comment by KasparoVy** [comment]
`Done` :thinking:

File: code/game/machinery/computer/buildandrepair.dm [hunk 1]
```diff
@@ -219,6 +219,9 @@
 /obj/item/circuitboard/prisoner
 	name = "Circuit board (Prisoner Management)"
 	build_path = /obj/machinery/computer/prisoner
+/obj/item/circuitboard/brigcells
```
Comment (variableundefined) [inline code/game/machinery/computer/buildandrepair.dm]: Spacing between this and previous line please

Comment (AffectedArc07) [reply code/game/machinery/computer/buildandrepair.dm]: @variableundefined It’s like that for the whole file. Unless you want me to space the **whole** file it’s staying as it is

**Comment by AffectedArc07** [comment]
Ok to resolve my conflicts I have to put in the shuttle navigation consoles.

Which arent in my local code.

What do I do

**Comment by AffectedArc07** [comment]
Oh I love conflicts.
*just shoot me*

**Comment by AffectedArc07** [comment]
Deconflicted @variableundefined

**Comment by variableundefined** [comment]
What happened to paradise.dme

**Comment by AffectedArc07** [comment]
@variableundefined I dont know ok!

**Comment by variableundefined** [comment]
That has to be fixed before I can let this go in or review it.

**Comment by AffectedArc07** [comment]
@variableundefined Ready for review, and fixed the `.dme` issue

**Comment by AffectedArc07** [comment]
Travis worked for all except delta.
*reeees in distance*

**Pull request merged**